client/.gitignore

```diff
@@ -1,5 +1,8 @@
 # See http://help.github.com/ignore-files/ for more about ignoring files.
 
+# Closed-Source Fonts that must be copied in
+DarlingModernRegular.woff
```

Comment:
Do you just have this in the client root?

Reply:
nah but I'll make it more clear